### PR TITLE
[TIMOB-24988] Android: Add Ti.Network.Cookie isValid()

### PR DESCRIPTION
**Pull request by m1ga**
**JIRA:** https://jira.appcelerator.org/browse/TIMOB-24988

Parity with iOS

Test code
```
var cookie1 = Ti.Network.createCookie({
	domain: 'example.com',
	name: 'test_cookie',
	value: '12',
	path: '/'
});

var cookie2 = Ti.Network.createCookie({
	name: 'test_cookie',
	value: '12',
	path: '/'
});

console.log("is cookie valid: "+ cookie1.isValid());
console.log("is cookie valid: "+ cookie2.isValid());

```

### CODE REVIEW

File: android/modules/network/src/java/ti/modules/titanium/network/CookieProxy.java [hunk 1]
```diff
@@ -159,9 +159,19 @@ public String getName()
 		return TiConvert.toString(getProperty(TiC.PROPERTY_NAME));
 	}
 
+	@Kroll.method
+	public boolean isValid()
+	{
+		String name = TiConvert.toString(getProperty(TiC.PROPERTY_NAME), "");
```
Comment (garymathews) [inline android/modules/network/src/java/ti/modules/titanium/network/CookieProxy.java]: No need to set a default value in this case:
```Java
String name = TiConvert.toString(getProperty(TiC.PROPERTY_NAME));
String value = TiConvert.toString(getProperty(TiC.PROPERTY_VALUE));
String path = TiConvert.toString(getProperty(TiC.PROPERTY_PATH));
String domain = TiConvert.toString(getProperty(TiC.PROPERTY_DOMAIN));
return (name != null && value != null && path != null && domain != null);
```

**Review by garymathews — CHANGES_REQUESTED** [review]

**Review by garymathews — COMMENTED** [review]
CR: **PASS**

**Review by garymathews — APPROVED** [review]
CR: **PASS**

**Comment by hansemannn** [comment]
@m1ga Can you provide a 6_2_X PR as well? Currently this would only go into 7.0.0

**Comment by lokeshchdhry** [comment]
FR Passed.

`isValid()` works as expected.

Studio Ver:     4.9.1.201707200100
SDK Ver:        7.0.0 local build
OS Ver:         10.12.3
Xcode Ver:      Xcode 8.3.3
Appc NPM:       4.2.9
Appc CLI:       6.2.2
Ti CLI Ver:     5.0.14
Alloy Ver:      1.9.11
Node Ver:       6.10.1
Java Ver:       1.8.0_101
Devices:        Android 6.0

**Pull request merged**